lib/internal/main/repl.js

```diff
@@ -19,38 +19,44 @@ prepareMainThreadExecution();
 
 markBootstrapComplete();
 
-// --input-type flag not supported in REPL
-if (getOptionValue('--input-type')) {
-  // If we can't write to stderr, we'd like to make this a noop,
-  // so use console.error.
-  console.error('Cannot specify --input-type for REPL');
-  process.exit(1);
-}
+if (process.env.NODE_EXTERNAL_REPL_MODULE) {
+  require('internal/modules/cjs/loader')
+    .Module
+    ._load(process.env.NODE_EXTERNAL_REPL_MODULE, undefined, true);
```

Comment:
Is it possible to support multiple modules to be loaded here if we set NODE_EXTERNAL_REPL_MODULE to multiple node modules ?

Reply:
like comma separated or something? you could just pass a file that has multiple require calls in it.

Reply:
i have a deep moral aversion to using `return` at the top level, do you mind much if it is kept this way?

Reply:
No, it's OK to keep it (I just think it improves the readability by returning early).